uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -0,0 +1,122 @@
+<div id="crm-contact-actions-list">
+  <div id="bootstrap-theme">
+    <div class="row">
+      <div class="col-md-4 contact_action_menu__user_info">
```

Comment:
All custom classes should have `crm` prefix for admin, and `chr` for SSP.
So it should be `crm_contact_action_menu__user_info`. Please check other instances.